packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.integration.test.tsx

```diff
@@ -161,14 +162,7 @@ describe('TableColumnSchemaEditor', () => {
     })
 
     // Add a column
-    const addColumnButton = await screen.findByRole('button', {
-      name: 'Add Column',
-    })
-
-    await user.click(addColumnButton)
-    const nameFields = await screen.findAllByLabelText('Name')
-    await user.type(nameFields[nameFields.length - 1], 'newColumnName')
-
+    await addColumnModelToForm('newColumnName', user)
```

Comment:
Move this interaction to a utility that we will reuse in the CreateTableViewWizard tests